File: frontend/src/ts/ChannelParam.ts
export interface ChannelParam {
    appName: string
    channelId: string
    channelName: string
    channelDesc: string
    version: string
    //0：未选中，1：待打包，2：打包中，3：打包完成，4：打包失败
    status: number
    progress: number
    isChecked: boolean
    packageName: string

}

File: go/channelParam.go
package _go

type ChannelParam struct {
	AppName     string
	ChannelId   string
	ChannelName string
	ChannelDesc string
	Version     string
	//打包状态：未开始1，打包中2，打包成功0，打包失败-1
	status      int
	Progress    int
	IsChecked   bool
	PackageName string
}
